offline/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
why is this here? You have a file in that directory called offline.html but you are also telling git to ignore it?